packages/@okta/vuepress-site/docs/guides/protect-your-api/main/index.md

```diff
@@ -151,8 +155,12 @@ The expected results are as follows, provided you followed the instructions in t
 
 If you are using Postman, your GET call and response should look something like this:
 
+<div class="full border">
```

Comment:
Re-size to 3/4 and also do some data cleanup in this image.

Reply:
done, same as above.